docs/cmd/kn_container_add.md

```diff
@@ -0,0 +1,59 @@
+## kn container add
+
+Add a container
+
+```
+kn container add NAME
+```
+
+### Examples
+
+```
+
+  The 'container add' represents utility command that prints YAML container spec to standard output. It's useful for
```

Comment:
Could we mark the command as "experimental" ? Still not sure if people like the UX, so this could be a hint to them.